src/pages/DashboardPage/DashboardProfilePage/DashboardProfilePage.css

```diff
@@ -0,0 +1,21 @@
+.profile-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-height: 600px;
```

Comment:
Min-height based on being smaller than the pixels required for the iPhone X, will need to do more when adjusting design soon